spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -43,6 +45,78 @@ module CCMS # rubocop:disable Metrics/ModuleLength
         end
       end
 
+      context 'CLIENT_ELIGIBILITY and PUI_CLIENT_ELIGIBILITY' do
+        context 'eligible' do
+          let!(:cfe_result) { create :cfe_result, submission: cfe_submission }
+          it 'returns In Scope' do
+            block = XmlExtractor.call(xml, :global_means, 'CLIENT_ELIGIBILITY')
+            expect(block).to have_text_response 'In Scope'
+            block = XmlExtractor.call(xml, :global_means, 'PUI_CLIENT_ELIGIBILITY')
+            expect(block).to have_text_response 'In Scope'
+          end
```

Comment:
Just a suggestion, but there's a pattern used in other specs where we're testing multiple attributes which you could use here. Something like:

```
        it 'returns In Scope' do
          attributes = %w[CLIENT_ELIGIBILITY PUI_CLIENT_ELIGIBILITY]
          attributes.each do |attribute|
            block = XmlExtractor.call(xml, :global_means, attribute)
            expect(block).to have_text_response 'In Scope'
          end
        end
```

It's a little dry-er. It could be used for the `not_eligible` and `contribution_required ` contexts too